OtripleS.Web.Api.Tests.Acceptance/Brokers/OtripleSApiBroker.Teacher.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using OtripleS.Web.Api.Models.Teachers;
+
+namespace OtripleS.Web.Api.Tests.Acceptance.Brokers
+{
+    public partial class OtripleSApiBroker
+    {
+        private const string TeachersRelativeUrl = "api/teachers";
+
+        public async ValueTask<Teacher> PostTeacherAsync(Teacher student) =>
```

Comment:
You need to rename parameters to be teacher instead of student

Reply:
good catch done